releasenotes/notes/new-deprecation-utilities-066aff05e221d7b1.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
I named the release note this because I think it makes sense to include https://github.com/Qiskit/qiskit-terra/pull/9676 in the same note. (That PR will be unblocked after this lands 🎉)

Reply:
the release note name doesn't actually appear publicly anywhere (technically it appears as a comment in the rst output by reno), so we've got some pretty funky ones historically.  It _technically_ defines the sort order of the output though haha